### PR TITLE
Move some packages into dev deps

### PR DESCRIPTION
**Pull request by jwolski**
@Raynos

### CODE REVIEW

**Comment by Raynos** [comment]
lgtm.

File: package.json [hunk 1]
```diff
@@ -19,19 +19,17 @@
   "dependencies": {
     "async": "^0.9.0",
```
Comment (Raynos) [inline package.json]: Get rid of async, no longer used.

File: package.json [hunk 2]
```diff
@@ -19,19 +19,17 @@
...
-    "cli-color": "^0.3.2",
-    "commander": "^2.6.0",
-    "debuglog": "^1.0.1",
     "error": "^5.0.0",
-    "express": "^4.9.5",
     "farmhash": "^0.2.0",
     "metrics": "^0.1.8",
     "readable-stream": "^1.0.33",
-    "time-mock": "^0.1.2",
     "underscore": "^1.5.2"
```
Comment (Raynos) [inline package.json]: We can refactor this dependency away.

File: package.json [hunk 3]
```diff
@@ -19,19 +19,17 @@
   "dependencies": {
     "async": "^0.9.0",
     "body": "^4.5.0",
```
Comment (Raynos) [inline package.json]: we can get this dependency replace with something lighter or something inline.

Comment (Raynos) [inline package.json]: I should probably update it to use raw-body instead.

File: package.json [hunk 4]
```diff
@@ -19,19 +19,17 @@
...
-    "cli-color": "^0.3.2",
-    "commander": "^2.6.0",
-    "debuglog": "^1.0.1",
     "error": "^5.0.0",
-    "express": "^4.9.5",
     "farmhash": "^0.2.0",
     "metrics": "^0.1.8",
     "readable-stream": "^1.0.33",
-    "time-mock": "^0.1.2",
```
Comment (Raynos) [inline package.json]: Is this not being used in tests? Shouldnt this be a dev dependency ?

Comment (jwolski) [inline package.json]: it's already listed as a `devDependency` so it was redundant in that sense

File: package.json (hunk 2 above)
Comment (jwolski) [inline package.json]: soon

**Pull request merged**